### PR TITLE
Book: rustup.sh -> rustup.rs

### LINKED ISSUE #35653: In the current book, recommend rustup.rs over rustup.sh
Specifically at https://doc.rust-lang.org/book/getting-started.html#installing-on-linux-or-mac, and recommend rustup.rs for Windows too, right?

We've [switched the new book to do this already](https://github.com/rust-lang/book/commit/8af84e7514be2a3aa72e3a421925cf68cb73f342), but in the meantime, the old book should recommend the new recommended toolchain :)

If it's too soon to do this because rustup.rs isn't stable enough yet, consider this a reminder issue to make this change once it is :)

### PR DESCRIPTION
**Pull request by steveklabnik**
Fixes #35653

cc https://github.com/rust-lang/rust-www/pull/621

### CODE REVIEW

**Comment by retep998** [comment]
This only covers installing rustup from a bash terminal. It doesn't cover installing rustup from either powershell or cmd. Granted, cmd doesn't really have a way to download files, but powershell totally does: `Invoke-WebRequest`. At the very least there should still be a section describing how to download and install the rustup.rs installer on Windows.

**Comment by steveklabnik** [comment]
@retep998 do you have the time to give me some phrasing of how you'd like it to appear? My only experience has been with bash.

**Comment by retep998** [comment]
cc @brson

**Comment by carols10cents** [comment]
@steveklabnik The new book is [already using rustup.rs](https://github.com/rust-lang/book/blob/3c38b59ab66641c5d118e57cf507801f46921f94/src/ch01-01-installation.md).

**Comment by steveklabnik** [comment]
Yeah, I was thinking of the Windows changes that happen.

**Comment by ollie27** [comment]
Might it be better to actually include less information in the book and basically just link to https://www.rust-lang.org/install.html so there is a single source of truth?

**Comment by steveklabnik** [comment]
@ollie27 This text is included in the offline documentation, and the website isn't. So I still like having it in the book. Most books have text in them about installing things, even if there's a site.

File: src/doc/book/getting-started.md [hunk 1]
```diff
@@ -106,9 +94,7 @@ resources include [the user’s forum][users] and [Stack Overflow][stackoverflow
 [stackoverflow]: http://stackoverflow.com/questions/tagged/rust
 
 This installer also installs a copy of the documentation locally, so we can
-read it offline. On UNIX systems, `/usr/local/share/doc/rust` is the location.
-On Windows, it's in a `share/doc` directory, inside the directory to which Rust
-was installed.
+read it offline. It's only a `rustup doc` away!
```
Comment (steveklabnik) [inline src/doc/book/getting-started.md]: Let's see if that behavior actually survives; seems fairly controversial.

**Comment by brson** [comment]
Looks good.

I agree it should say something about windows.

One option is telling them to navigate to `https://www.rust-lang.org/install.html` is probably best. You might present it like "Installing Rust is really simple. The instructions are on the Rust website (link), but if you are on Linux and Mac you can just type:"

It might also be worthwhile to present the download link directly in the text.

I guess for now I'd suggest keeping the two sections, one for Linux/Mac and one for Windows. In the windows section link directly to `htts://win.rustup.rs` saying like "On Windows download and run rustup-init.exe", linking to it.

I do think a link to install.html is warranted still since there's more to learn there. "For other installation options, see the install page on the Rust website". etc.

I'll also try to get a [powershell one-liner](https://github.com/rust-lang-nursery/rustup.rs/issues/829) working. If we had that we might present the curl instructions, the powershell instructions, and then "for other installation methods see the website".

**Comment by brson** [comment]
Closing in favor of https://github.com/rust-lang/rust/pull/38122

**Pull request closed without merging**